mt-st.bash_completion

```diff
@@ -22,8 +23,19 @@ _mt () {
             COMPREPLY=($(compgen -W "$devs" -- "$cur"))
             return
             ;;
+	stsetoptions)
+	    # show list of stoptions
+	    COMPREPLY=($(compgen -W "$stoptions" -- "$cur"))
+            return
+            ;;
     esac
 
+    # if "$prev" is a substring of "$stoptions" show more "$stoptions"
```

Comment:
This section gets invoked if the current command is:
` mt -f /dev/nst0 stsetoptions ${stoption} `

Reply:
This works as well as it could. A cleaner option woul be to test `stsetoptions*<space>`, rather than just prev member of stoptions, since currently this is broken for completion if one writes `mt no-wait` (just the option, without the `stsetoptions` command).

But this is better than the current situation, so I'll accept as-is.